requirements/constraints.txt

```diff
@@ -371,3 +377,6 @@ Werkzeug==0.15.5 \
 # django-debug-toolbar
 sqlparse==0.1.19 \
     --hash=sha256:d896be1a1c7f24bffe08d7a64e6f0176b260e281c5f3685afe7826f8bada4ee8
+Jinja==1.2 \
```

Comment:
Is this really a requirement of `django-debug-toolbar`?

Reply:
It's not, I'm removing it right now.